functionary/train/train_lora.py

```diff
@@ -0,0 +1,396 @@
+import json
+import math
+import pathlib
+import random
+from dataclasses import dataclass, field
+from typing import Dict, Optional
+
+import torch
+import torch.distributed
+from torch.nn import CrossEntropyLoss
+import sys
+import os
+
+os.environ["WANDB_LOG_MODEL"] = "all"
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../.."))
```

Comment:
Why is this needed ? We should use module running like "python3 -m functionary.train.XXX" and it shouldnt be needed

Reply:
Oh this is because we are using deepspeed so the command should be:
deepspeed functionary/train/train_lora.py
If we run: 
deepspeed -m functionary.train.train_lora